template1/index.html

```diff
@@ -5,7 +5,9 @@
     <title>Template 1</title>
     <link rel="stylesheet" href="style.css">
   </head>
-  <body>
+  <body style="background: url('https://wallpaper-mania.com/wp-content/uploads/2018/09/High_resolution_wallpaper_background_ID_77701744932.jpg');
+              background-repeat: no-repeat;
+              background-size: cover">-->
```

Comment:
For this project, you shouldn't need to edit index.html at all. All changes to CSS should be done in style.css.

If you want to change the background of the body, use `body { /*declaration*/}`. If you want to change the background of a specific id, use `body { /*declaration */ }`

Make sure you are checking the license of assets you use from other websites. Are they royalty free, or do you need to pay for them?